Automattic/wp-calypso
# Plugins update manager: Improve UX

Related to https://github.com/Automattic/dotcom-forge/issues/5544

## Proposed Changes

* Optimistically update the cache on create, update, and delete mutations
* Rename query and mutation hooks to follow the same convention
* Adapt submit button busy states

Known issues:
- Submit button color https://github.com/Automattic/wp-calypso/issues/88231
- Schedule name: currently, we consume the entered name, which is replaced once we get the server's info (currently "jetpack_scheduled_update") https://github.com/Automattic/wp-calypso/issues/88232

## Testing Instructions



* Go to `/plugins/scheduled-updates/{ATOMIC SITE}`
* Check CRUD actions and UX around

![Screen Capture on 2024-03-06 at 13-50-32](https://github.com/Automattic/wp-calypso/assets/1241413/374912e5-a992-4de2-97d7-f45dca7c719b)



## Pre-merge Checklist
